modules/ROOT/pages/6.5-release-notes.adoc

```diff
@@ -30,6 +30,23 @@
 
 The following premium plugin updates were released alongside {productname} 6.5.
 
+=== Enhanced Media Embed 3.1.2
+
+The {productname} 6.5 release includes an accompanying release of the **Enhanced Media Embed** premium plugin.
+
+**Enhanced Media Embed** 3.1.2 includes the following fixes, additions, and improvements.
+
+=== Allow a media embed element to be correctly resized when using the `media` plugin dialog by converting the responsive media embed to a standalone iframe.
+//#TINY-8714
+
+In previous versions of {productname}, the Mediaembed plugin and the media plugin encountered several issues when used together. The Mediaembed plugin would wrap `<iframes>` with a responsive wrapper that relies on large padding for the height. However, when content created with the Mediaembed plugin is inserted using the media plugin dialog, which targets `"ephox-embed-iri"` divs, the width and height are mapped to the wrapper's `max-width` and `max-height` properties. Unfortunately, the height is primarily determined by padding, rendering the `max-height` property ineffective.
+
+For additional details, please see https://www.benmarshall.me/responsive-iframes[benmarshall.me/responsive-iframes].
+
+Another problem that arised when inserting embedded content via the mediaembed plugin, was that the element may wave appeared stretched while lacking proportional sizing. This occured because the height was not explicitly set, which leaft it to be determined by the content within the `<iframe>`.
```

Comment:
```suggestion
Another problem that arised when inserting embedded content via the mediaembed plugin, was that the element may have appeared stretched while lacking proportional sizing. This occurred because the height was not explicitly set, which left it to be determined by the content within the `<iframe>`.
```